meta-resin-common/recipes-support/resin-mounts/resin-mounts/var-lib-dropbear.service

```diff
@@ -0,0 +1,18 @@
+[Unit]
+Description=Dropbear Host Key Mount Point
+Requires=mnt-conf.mount
+After=mnt-conf.mount
+Before=dropbearkey.service
+DefaultDependencies=false
+ConditionPathIsReadWrite=/mnt/conf/root-overlay/var/lib/dropbear
+
+[Service]
+Type=oneshot
+RemainAfterExit=Yes
+StandardOutput=syslog
+TimeoutSec=0
+ExecStart=/sbin/mount-copybind /mnt/conf/root-overlay/var/lib/dropbear /var/lib/dropbear
```

Comment:
It will get messy. How would you name that directory (assuming you want to keep it in /var/lib)?